config.vim

```diff
@@ -344,7 +367,7 @@ nmap ,D :tab split<CR>:call jedi#goto()<CR>
 " Ack.vim ------------------------------
 
 " mappings
-nmap ,r :Ack 
+nmap ,r :Ack
```

Comment:
this space is meaningfull and needed, the mapping leaves the cursor at the end of the command, with a space in between, so the user can star typing the search text